cita-executor/core/src/core_types/common.rs

```diff
@@ -0,0 +1,245 @@
+// FIXME: Please careful about the default hashlib, use hashlib-keccak for now.
+use std::fmt;
+
+use numext_fixed_hash::H160;
+pub use numext_fixed_uint::U256;
+use rlp::{Encodable, RlpStream};
+use serde::{Serialize, Serializer};
+#[cfg(feature = "hashlib-sha3")]
+use sha3::{Digest, Sha3_256};
+
+use crate::core_types::errors::TypesError;
+
+const ADDRESS_LEN: usize = 20;
+const HASH_LEN: usize = 32;
+
+pub type Balance = U256;
+pub type H256 = numext_fixed_hash::H256;
+
+/// Address represents the 20 byte address of an cita account.
+#[derive(Default, Clone, PartialEq, Eq, Hash)]
+pub struct Address(H160);
```

Comment:
why not just use ethereum_types ?

Reply:
Maybe later. There are some methods in this struct, if there is not logic call this methods, I will replace it to ethereum_types.